include/swift/ClangImporter/ClangImporter.h

```diff
@@ -242,8 +242,7 @@ class ClangImporter final : public ClangModuleLoader {
   ///
   /// \sa clang::GeneratePCHAction
   bool emitBridgingPCH(StringRef headerPath,
-                       StringRef outputPCHPath,
-                       clang::DiagnosticConsumer *Diags = nullptr);
```

Comment:
The change to Diags is the only bit I'm not entirely clear on; the rest looks fine.